Resources/doc/first_steps.rst

```diff
@@ -438,7 +438,64 @@ You can use this new method just like the default finder methods of the reposito
     When using a custom repository class, you still have access to the default
     finder methods such as ``find()`` and ``findAll()``.
 
+Service Repositories
+~~~~~~~~~~~~~~~~~~~~
+
+In the previous section, you learnt how to create custom repository classes and how
+to get them using ``DocumentManager``. Another way of obtaining a repository instance
+is to use the repository as a service and inject it as a dependency into other services.
+
+.. code-block:: php
+
+    // src/Acme/StoreBundle/Repository/ProductRepository.php
+    namespace Acme\StoreBundle\Repository;
```

Comment:
Since Symfony no longer recommends writing bundles for your own code, maybe we should update this. Feel free to create a separate ticket for this if it would generate additional changes in this file to be consistent.

Reply:
There's #536 to track bundle removal. Here I made the paths consistent with other stuff in the chapter

Reply:
Sounds good to me.